source/NationalInstruments/jquery.flot.intensitygraph.js

```diff
@@ -189,6 +189,9 @@ THE SOFTWARE.
 
                 for (let x = 0; x < dataWidth; x++) {
                     for (let y = 0; y < dataHeight; y++) {
+                        if (Number.isNaN(seriesData[x][y])) {
```

Comment:
Why is it not valid to find NaNs? I could see how someone might want to see this interactively.

Reply:
NaNs are not drawn on intensity graphs (i.e. there's no color for a NaN value), so I think this would be interpreted as "no value". I'm not 100% sure about this though, I think it's worth a discussion. Maybe @andrewdove1 could speak to how NaN values should be interpreted on an intensity graph.

Reply:
So, to be clear, yes, I expect NaN values to not be drawn, but that doesn't mean hovering over the area of a NaN value should not report that it is, in fact, a NaN value. In fact, a user could be horribly confused if you hover over a part of the intensity graph that looks like a value (because whatever the default color of the plot area is matches a color on the color scale, and all of the sudden nothing is reported. It might be more useful if we explicitly reported "this value is NaN", instead of nothing.

Reply:
I agree that reporting NaN values on hover makes sense.